From e6dfbe67f94d60b889f3031ebb644ed262d7fe35 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Sat, 16 Apr 2022 14:43:17 +0200
Subject: [PATCH] test: install /usr/libexec/vi as well

since `/bin/vi` (at least on Fedora) is a shell wrapper which runs
either `/bin/vim` or `/usr/libexec/vi` based on availability.

(cherry picked from commit 8afe2f53b25ca99bc5bd1ec0c5dff7e183712577)

Related: #2087652
---
 test/test-functions | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/test-functions b/test/test-functions
index 1306dcf260..9e171cba30 100644
--- a/test/test-functions
+++ b/test/test-functions
@@ -225,6 +225,7 @@ DEBUGTOOLS=(
     stty
     tty
     vi
+    /usr/libexec/vi
 )
 
 is_built_with_asan() {
